README_first.ipynb

```diff
@@ -1,60 +1,109 @@
 {
```

Comment:
<p>Wouldn't call it advanced would call it, "Useful tips" or "good to know", maybe not nested. The reason for my thought process here is that we need the users to know this exists so they can solve their problems.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/graphcore/Graphcore-HuggingFace-fork/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>